app/models/miq_report.rb

```diff
@@ -262,7 +262,7 @@ def group_description=(_group_description)
   def columns_for_sorting(columns)
     columns = columns.split(",") if columns && columns.kind_of?(String)
 
-    columns || sortby || col_order
+    columns || sortby || cols
```

Comment:
I would prefer to keep this as is.
Seems better to sort by columns that are actually present in the report